LaTeX/tests/syntax_test_latex.tex

```diff
@@ -334,11 +334,11 @@ \subsubsection{name}
 %           ^ support.function
 %
 
-\parencite[Propositon~1]{Ref}
+\parencite[Proposition~1]{Ref}
```

Comment:
I'm not quite sure what this is doing... So, I'm hoping this is ok, but less certain than most of the other corrections here...